vmaas_sync/advisory_sync.go

```diff
@@ -53,6 +56,11 @@ func getAdvisoryTypes() (map[string]int, error) {
 	for _, t := range advisoryTypesArr {
 		advisoryTypes[strings.ToLower(t.Name)] = t.ID
 	}
+
+	for from, to := range advisoryTypeRemap {
+		advisoryTypes[from] = advisoryTypes[to]
+	}
+
```

Comment:
All these 8 new lines can be replaced by:
~~~
advisoryTypes["newpackage"] = advisoryTypes["enhancement"]
~~~

Reply:
I'd rather have the var on the top of the file. Having constants somewhere in  the code gets unreadable really fast.

Reply:
I think that the line I suggested and explaining comment would be enough, but as you wish. So please at least add SPM-XYZ to PR title to get the link in Jira.

Reply:
... and to the commit message